sympy/solvers/tests/test_ode.py

```diff
@@ -663,6 +664,9 @@ def test_checkodesol():
     eqn = Eq(Derivative(x*Derivative(f(x), x), x)/x, exp(x))
     sol = Eq(f(x), C1 + C2*log(x) + exp(x) - Ei(x))
     assert checkodesol(eqn, sol, order=2, solve_for_func=False)[0]
+    eq = x**2*(f(x).diff(x, 2)) + x*(f(x).diff(x)) + (2*x**2 +25)*f(x)
+    sol = Eq(f(x), C1*besselj(5*I, sqrt(2)*x) + C2*bessely(5*I, sqrt(2)*x))
+    assert checkodesol(eq, sol)
```

Comment:
This assert will still pass if checkodesol returns `(False, expr)`. Use instead:
```
assert checkodesol(eq, sol) == (True, 0)
```

Reply:
sorry